core/src/main/web/app/mainapp/mainapp.js

```diff
@@ -729,6 +729,28 @@
                 }
               }
             },
+            saveNotebookAndClose: function() {
+              saveStart();
+              bkFileManipulation.saveNotebook(saveFailed).then(
+                  function(ret){
+                    //bkSessionManager.close();
+                    //bkCoreManager.gotoControlPanel();
+                    
+                    if (bkEvaluateJobManager.isAnyInProgress() ) {
```

Comment:
Because "closeNotebook" tries to save notebook. 
All after save actions take place in "saveDone" (marked us saved).
I do not call "saveDone" because if I call it I need to call "scope.apply", but I do not need this scope at all.
So if I call "closeNotebook" , application think that it is not saved and shows dialog where ofers to me save it.
The I decided to write my own "close".